### DISCUSSION TITLE
아래 단어에 대한 번역이 모호합니다.

### DISCUSSION THREAD

**Discussion by hoyajigi** [CS285-KOR/CS285_21_KOR_SUB#6] [번역 용어 토의&제안]
- overparameterized

> it's probably about the observation that any large heavily overparameterized model will discover features with these statistics

거대한 모델, 파라미터가 많은 모델인데 한국어로 한 단어는 없는거 같아서요

- spectrum dependent plasticity

> so percepts that anticipate reward become associated with similar firing patterns as the reward itself this is a known observation sometimes referred to as spectrum dependent plasticity

스펙트럼 의존 가소성 ???

왠만한 용어는 알겠는데 이건ㅠㅠ

**Comment by curieuxjy** [comment]
- overparameterized 는 
1. 그냥 번역하지 않고 영어로 `overparameterized`라고 해도 좋을 것 같고, 
2. 풀어서 번역해도 좋을 것 같아요. "A model having more parameters than can be estimated from the data."라는 뜻이니까 `데이터에서 추정할 수 있는 것보다 더 많은 모수를 가지고 있는 모델은...` 이라고 좋을 것 같습니다. 
3.  약간의 의역으로 `"너무 많은" 파라미터를 가지고 있는 모델`이라는 걸로 쓰면 "large heavily over-"라는 의미를 잘 살릴 수 있을 것 같습니다.

- spectrum dependent plasticity 저도 찾아보니 신경과학쪽에서도 쓰이는 용어처럼 보이는데 써주신 문장에서도 용어 정리 느낌으로 가는 것 같아서 그냥 영어로 쓰는 것도 좋을 것 같아요.
